cli/src/main.rs

```diff
@@ -108,13 +108,20 @@ async fn main() -> Result<()> {
             for (cluster_name, cluster) in &state.clusters {
                 for (backend_id, backend) in &cluster.backends {
                     if let Some((timestamp, state)) = backend.state_timestamp() {
-                        if state != BackendState::Swept {
+                        if state == BackendState::Swept || state == BackendState::Terminated {
+                            println!(
+                                "Removing backend {} from cluster {}, {} at {}",
+                                backend_id, cluster_name, state, timestamp
+                            );
+                        } else if timestamp < chrono::Utc::now() - chrono::Duration::hours(24) && (state != BackendState::Ready) {
+                            println!(
+                                "Removing abandoned backend {} from cluster {}, last update was {} at {}",
+                                backend_id, cluster_name, state, timestamp
+                            );
```

Comment:
Should we ensure those backend containers are no longer running by telling the drones to stop them and then set their statuses to `Lost`?

Reply:
That should really be handled by the drone itself. The platform currently has no way of telling the drone that a backend should be considered lost. This is more of a catch to ensure that if the drone is buggy we don't end up accumulating a bunch of dead state.

Reply:
Done.